[warn] Epoll ADD(1) on fd 0 failed.  Old events were 0; read change was 1 (add); write change was 0 (none): Operation not permitted
[warn] Epoll ADD(4) on fd 1 failed.  Old events were 0; read change was 0 (none); write change was 1 (add): Operation not permitted
[warn] Epoll ADD(4) on fd 2 failed.  Old events were 0; read change was 0 (none); write change was 1 (add): Operation not permitted
Traceback (most recent call last):
  File "/nfs/home/vpais/ThesisPython/./AuMieSphere/u_au_mie_scattering.py", line 985, in <module>
Traceback (most recent call last):
  File "/nfs/home/vpais/ThesisPython/./AuMieSphere/u_au_mie_scattering.py", line 985, in <module>
Traceback (most recent call last):
  File "/nfs/home/vpais/ThesisPython/./AuMieSphere/u_au_mie_scattering.py", line 985, in <module>
Traceback (most recent call last):
  File "/nfs/home/vpais/ThesisPython/./AuMieSphere/u_au_mie_scattering.py", line 985, in <module>
Traceback (most recent call last):
  File "/nfs/home/vpais/ThesisPython/./AuMieSphere/u_au_mie_scattering.py", line 985, in <module>
    main()
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1137, in __call__
Traceback (most recent call last):
  File "/nfs/home/vpais/ThesisPython/./AuMieSphere/u_au_mie_scattering.py", line 985, in <module>
Traceback (most recent call last):
  File "/nfs/home/vpais/ThesisPython/./AuMieSphere/u_au_mie_scattering.py", line 985, in <module>
    main()
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1137, in __call__
Traceback (most recent call last):
  File "/nfs/home/vpais/ThesisPython/./AuMieSphere/u_au_mie_scattering.py", line 985, in <module>
    main()
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1137, in __call__
Traceback (most recent call last):
  File "/nfs/home/vpais/ThesisPython/./AuMieSphere/u_au_mie_scattering.py", line 985, in <module>
    main()
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1137, in __call__
    main()
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1137, in __call__
    main()
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1062, in main
    main()
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1062, in main
    main()
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1062, in main
    main()
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1062, in main
    return self.main(*args, **kwargs)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1062, in main
    return self.main(*args, **kwargs)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return self.main(*args, **kwargs)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return self.main(*args, **kwargs)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    rv = self.invoke(ctx)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return self.main(*args, **kwargs)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 763, in invoke
    rv = self.invoke(ctx)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    rv = self.invoke(ctx)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 763, in invoke
    rv = self.invoke(ctx)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 763, in invoke
    rv = self.invoke(ctx)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/nfs/home/vpais/ThesisPython/./AuMieSphere/u_au_mie_scattering.py", line 281, in main
    return ctx.invoke(self.callback, **ctx.params)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 763, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/nfs/home/vpais/ThesisPython/./AuMieSphere/u_au_mie_scattering.py", line 281, in main
    return ctx.invoke(self.callback, **ctx.params)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 763, in invoke
    os.makedirs(path)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/os.py", line 225, in makedirs
    return ctx.invoke(self.callback, **ctx.params)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/nfs/home/vpais/ThesisPython/./AuMieSphere/u_au_mie_scattering.py", line 281, in main
    return __callback(*args, **kwargs)
  File "/nfs/home/vpais/ThesisPython/./AuMieSphere/u_au_mie_scattering.py", line 281, in main
    return __callback(*args, **kwargs)
  File "/nfs/home/vpais/ThesisPython/./AuMieSphere/u_au_mie_scattering.py", line 281, in main
    return __callback(*args, **kwargs)
  File "/nfs/home/vpais/ThesisPython/./AuMieSphere/u_au_mie_scattering.py", line 281, in main
    os.makedirs(path)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/os.py", line 225, in makedirs
    return ctx.invoke(self.callback, **ctx.params)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/site-packages/click/core.py", line 763, in invoke
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: '/nfs/home/vpais/ThesisResults/AuMieSphere/AuMie/15)TUPAC/1NodeTest/1NodeRes2'
    os.makedirs(path)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/os.py", line 225, in makedirs
    return __callback(*args, **kwargs)
  File "/nfs/home/vpais/ThesisPython/./AuMieSphere/u_au_mie_scattering.py", line 281, in main
    os.makedirs(path)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/os.py", line 225, in makedirs
    return __callback(*args, **kwargs)
  File "/nfs/home/vpais/ThesisPython/./AuMieSphere/u_au_mie_scattering.py", line 281, in main
    os.makedirs(path)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/os.py", line 225, in makedirs
    os.makedirs(path)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/os.py", line 225, in makedirs
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: '/nfs/home/vpais/ThesisResults/AuMieSphere/AuMie/15)TUPAC/1NodeTest/1NodeRes2'
    return __callback(*args, **kwargs)
  File "/nfs/home/vpais/ThesisPython/./AuMieSphere/u_au_mie_scattering.py", line 281, in main
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: '/nfs/home/vpais/ThesisResults/AuMieSphere/AuMie/15)TUPAC/1NodeTest/1NodeRes2'
    os.makedirs(path)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/os.py", line 225, in makedirs
    os.makedirs(path)
    mkdir(name, mode)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/os.py", line 225, in makedirs
FileExistsError: [Errno 17] File exists: '/nfs/home/vpais/ThesisResults/AuMieSphere/AuMie/15)TUPAC/1NodeTest/1NodeRes2'
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: '/nfs/home/vpais/ThesisResults/AuMieSphere/AuMie/15)TUPAC/1NodeTest/1NodeRes2'
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: '/nfs/home/vpais/ThesisResults/AuMieSphere/AuMie/15)TUPAC/1NodeTest/1NodeRes2'
    os.makedirs(path)
  File "/nfs/home/vpais/miniconda/envs/pmp/lib/python3.9/os.py", line 225, in makedirs
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: '/nfs/home/vpais/ThesisResults/AuMieSphere/AuMie/15)TUPAC/1NodeTest/1NodeRes2'
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: '/nfs/home/vpais/ThesisResults/AuMieSphere/AuMie/15)TUPAC/1NodeTest/1NodeRes2'
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: '/nfs/home/vpais/ThesisResults/AuMieSphere/AuMie/15)TUPAC/1NodeTest/1NodeRes2'
[warn] Epoll ADD(1) on fd 0 failed.  Old events were 0; read change was 1 (add); write change was 0 (none): Operation not permitted
[warn] Epoll ADD(4) on fd 1 failed.  Old events were 0; read change was 0 (none); write change was 1 (add): Operation not permitted
slurmstepd: error: *** JOB 380762 ON cnode42 CANCELLED AT 2021-07-25T17:33:11 ***
mpirun: Forwarding signal 18 to job
